src/Features/Blockcore.Features.Wallet/Interfaces/IAccountRoot.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using Blockcore.Features.Wallet.Database;
+using Blockcore.Networks;
+using NBitcoin;
+
+namespace Blockcore.Features.Wallet.Interfaces
+{
+    public interface IAccountRoot
```

Comment:
Out of curiosity was it necessary to break this to interfaces?

Reply:
This provides ability to substitute objects seamlessly and WalletManager and other infrastructure components use them in a usual way. Therefore saving a lot of effort writing in infrastructure if NormalWallet do something if MultisigWallet do something too. Also this actually makes a code more testable and easier to refactor.